lte/gateway/c/core/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -985,7 +985,8 @@ void mme_app_handle_delete_session_rsp(
     OAILOG_FUNC_OUT(LOG_MME_APP);
   }
 
-  if (ue_context_p->tau_accept_eps_ber_cntx_status) {
+  if (ue_context_p->tau_accept_eps_ber_cntx_status &&
+      ue_context_p->nb_delete_sessions) {
```

Comment:
Explicit check for no-zero value. Shouldn't the check happen earlier in this function call?
```suggestion
      ue_context_p->nb_delete_sessions > 0) {
```